tests/baselines/reference/arrowFunctionContexts.js

```diff
@@ -116,7 +116,8 @@ var Base = (function () {
 var Derived = (function (_super) {
     __extends(Derived, _super);
     function Derived() {
-        return _super.call(this, function () { return _this; }) || this;
+        var _this = _super.call(this, function () { return _this; }) || this;

```

Comment:
These baselines don't quite seem correct. Surely, _this should be declared and assigned to `this` first before the `super` call. That way, if the super call invokes the function, it'll still function as expected if the super does not return a new value for `this`.